blockchain/blockchain.go

```diff
@@ -1309,7 +1309,13 @@ func (bc *BlockChain) writeBlockLogsToRemoteCache(blockLogsKey []byte, receipts
 		return
 	}
 	// TODO-Klaytn-KES: refine this not to use trieNodeCache
-	bc.stateCache.TrieDB().TrieNodeCache().Set(blockLogsKey, encodedBlockLogs)
+	cache, ok := bc.stateCache.TrieDB().TrieNodeCache().(*statedb.HybridCache)
+	if !ok {
+		logger.Error("only HybridCache supports block logs writing",
+			"TrieNodeCacheType", reflect.TypeOf(bc.stateCache.TrieDB().TrieNodeCache()))
+	} else {
+		cache.Remote().Set(blockLogsKey, encodedBlockLogs)
+	}
```

Comment:
What do you think of defining `SetRemote` function in `TrieNodeCache`?
Hybrid, remote cache -> write only on remote cache
local cach -> do nothing or return an error

Reply:
@winnie-byun I think it's a good suggestion, but we need to consider other things also. After adopting your suggestion, `HybridCache.Set` sets remote cache async but `HybridCache.SetRemote` sets remote cache sync. I think it causes some misconceptions. Do you think using `HybridCache.SetRemoteSync` is better? Please, let me know if you have other opinions.

Reply:
I think it should be `HybridCache.SetRemote` and `HybridCache.SetRemoteAsync` to match the name with `RedisCache`. In this way, it would not be confusing.
Name the way you would not confused :)